src/main/java/com/iexec/core/task/TaskController.java

```diff
@@ -100,14 +100,20 @@ ReplicateModel buildReplicateModel(Replicate replicate) {
         return replicateModel;
     }
 
-    @GetMapping("/tasks/{chainTaskId}/stdout")
+    @GetMapping(path = {
+            "/tasks/{chainTaskId}/stdout",  // @Deprecated
+            "/tasks/{chainTaskId}/logs"
+    })
     public ResponseEntity<TaskStdout> getTaskStdout(@PathVariable("chainTaskId") String chainTaskId) {
```

Comment:
```
TaskLogs
getTaskLogs
ReplicateLogs
...
*Stdout -> *Logs
```
?

Reply:
We have some properties containing `stdout`:
```
stdout:
  purge-rate-in-days: ${IEXEC_STDOUT_PURGE_RATE_IN_DAYS:1}
  availability-period-in-days: ${IEXEC_STDOUT_AVAILABILITY_PERIOD_IN_DAYS:3}
```
I'm not sure we should update them as it could break external scripts/deployments.

Reply:
BTW, if we want to later add `stderr`, it may be clearer to keep `stdout` there :thinking:

Reply:
Considering it is not likely clients use yml values directly but use environment variables, we might try that kind of things (?): 
```
logs:
  purge-rate-in-days: ${IEXEC_STDOUT_PURGE_RATE_IN_DAYS:${IEXEC_LOGS_PURGE_RATE_IN_DAYS:1}}
```
About your comment on stderr, we should target
```
logs {
  stdout
  stderr
}
```
When talking about purge rate, etc, log policy will be the same for both stdout and stderr (since we'll play with a single `logs` object).

What do you think?

Reply:
Please see [Common commit](https://github.com/iExecBlockchainComputing/iexec-common/pull/326/commits/c171f751d38b879928839167df82adad4cbab3a9) and [Core commit](https://github.com/iExecBlockchainComputing/iexec-core/pull/504/commits/a05aeb4c01690740aba426c0841c1f9ad2d4b1a9).